### DISCUSSION TITLE
Having this General Meeting is Important!

### DISCUSSION THREAD

**Discussion by Rayccheal** [clentafrica/hr-people-operations#20] [Polls]
Dear Team, @clentafrica/wg-design @clentafrica/wg-marketing-comms @clentafrica/wg-hr @clentafrica/wg-health @clentafrica/wg-sustainability-esg @clentafrica/wg-research-and-data @clentafrica/wg-disability-gender-equality @clentafrica/wg-community-managers @clentafrica/wg-education @clentafrica/wg-web-dev @clentafrica/wg-energy @yigakpoa @Onodjayeke @EmmanetD 

I hope you are doing well today. 

I want to begin by expressing my gratitude to everyone who turned out for the General Meeting on Saturday, November 2nd, 2024. It is truly encouraging to see our shared passion for providing sustainable solutions to combat climate change in Africa. Thank you!

While I appreciate the attendance, only five people were present, and as you can agree, this is not sufficient to consider a general meeting successful. 

To reiterate the importance of our gathering, here are the key agendas we looked at discussing:

1. Familiarizing ourselves to build community
2. Understanding the organizational structure at CLENT Africa
3. Clarifying roles and responsibilities of team member
4. Integrating new volunteers and addressing their common questions
5. Discussing pain points and proposing solutions
6. Presenting upcoming team projects by team leads
7. Question and answer
8. Thought sharing, etc.

Considering the importance of these topics and the need to address them, it is essential we reconvene. 

I understand that scheduling can be challenging due to other commitments, so we will be conducting a poll to determine the most convenient time for everyone. Please take a moment to select your preferred date and time, and confirm your local time zone [here](https://www.worldtimebuddy.com/) before making your choice.

PS: All teams are expected to have their usual team meeting this week as discussed in the last team leads meeting as well, as this will greatly inform the rest of the agenda we will be treating in this general meeting. You can get clarifications from the meeting notes [here](https://docs.google.com/document/d/1s8N5hq8TyYCmcc4EPkbjx4hqfBpVRFnU8wL3PVJGX7M/edit?usp=sharing)

Finally, you can also share your thoughts and let us know other things that could be a hindrance to the proposed meetings using the comments section.

Thank you for your understanding and collaboration. I look forward to your response.

Best regards,
Racheal
HR Team Lead

**Comment by emylajem** [comment]
Good morning Raycheal
I am so sorry i couldn't attend the Saturday general meeting. I had some
emergencies. Please after careful thought I would love to be a team lead in
***@***.***

**Reply by Rayccheal** [reply]
Hi @emylajem,

Thank you so much for your interest in becoming a team lead at CLENT Africa; it’s greatly appreciated!

I would like to know if you have completed the onboarding process for this role, whether you were interviewed, and how active you have been in your current team.

Looking forward to your response!

**Reply by emylajem** [reply]
I have not been interviewed neither have i been onboarded. I want to focus on a team and start work immediately.

**Reply by yigakpoa** [reply]
Thank you @Rayccheal @EmmanetD & @emylajem 
for the great conversations on this. 

@emylajem I would just like to share that we're building using open source so please feel free to start contributing to the problem that directly appeals to you. Only thing is as an organization, we don't think contributors should be burnt out with too many teams. I hope this helps.

**Comment by emylajem** [comment]
Can someone be in more than one team at a time?

**Reply by Rayccheal** [reply]
Hi Emmanuel,

Thank you for your feedback!

I understand that your skills may enable you to contribute to multiple teams. However, at CLENT Africa, **it is highly encouraged that you focus on one team at a time**. This approach helps prevent overwhelm and allows you to deliver optimal results in your role.

Thank you for your understanding!

PS: You should have communicated with Damilare @EmmanetD by now as was discussed in the previous team leads meeting. Have you not set up your Discord yet?

Racheal

**Reply by emylajem** [reply]
Yes i agree with you on the need to belong to a team at a time. I thought Emmanuel Damilare was going to send me an email message, was that not the reason he requested for my email?

**Reply by EmmanetD** [reply]
Hello @emylajem, 

The CLENT Africa platform includes GitHub and Discord. You have been added to the team you indicated on GitHub: HR (https://github.com/orgs/clentafrica/people?query=Emmanuel+alajemba), and you should have received a notification to that effect. Additionally, a Discord link was shared during the call with you and another member. The other member has responded and now has access to the Discord platform. However, we are still awaiting your response (you are suppose to join the discord platform via the link!).

Thanks.

**Comment by Rayccheal** [comment]
Meanwhile, Kindly vote in the poll above @emylajem 

Thank you!

**Comment by 254white** [comment]
Hello Rayccheal, do you have a moment to talk? I'm unable to contact you
privately

Regards,

**Reply by Rayccheal** [reply]
Hi @254white,

Thank you for your feedback!

Are you a team lead? If so, you can join the team lead group [here](https://discord.gg/sdUvFnaS), where pressing questions and inquiries are addressed promptly. If you’re not a team lead, please let me know which team you belong to so I can reach out to you through your team’s group.

In the meantime, could you please take a moment to vote in the poll above?

Thank you as I look forward to hearing from you soonest. 

Best regards,
Racheal
